playbooks/common/openshift-cluster/config.yml

```diff
@@ -60,3 +60,7 @@
 - include: openshift_hosted.yml
   tags:
   - hosted
+
+- include: reset_excluder.yml
```

Comment:
The config is run during the installation only so the excluders are enabled at the end of the installation only.
During the installation I am assuming the cluster is vanilla new and without any excluder modifications (basically empty exclude item in ``/etc/yum.conf``. So it is ok to skip the status check. So basically it does not matter how many times we disable and re-enable the excluders. Cause the excluders must be to make sure affected packages are not updated out of installation/upgrade.

On the other hand, during the upgrade we have to disable and re-enable the excluders only once. Since disabling them means clearing the exclude item in ``/etc/yum.conf``. Which can be modified by a user (which is checked by the status command). So we should enable only those excluders that has not been changed by the user. Excluding docker cause we always want the docker excluder to take affect in order to safe the cluster from breaking apart during different docker version.